dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/DhisTest.java

```diff
@@ -51,7 +53,8 @@
  * @author Lars Helge Overland
  */
 @RunWith( SpringRunner.class )
-@ContextConfiguration( locations = { "classpath*:/META-INF/dhis/beans.xml" } )
+@ContextConfiguration( classes = { UnitTestConfiguration.class } )
+@ActiveProfiles("test")

```

Comment:
Can @ActiveProfiles("test") be moved to DhisConvenienceTest. If its possible, then it wouldnt be required to annotate neither of DhisTest or DhisSpringTest.

Reply:
Yap, that makes sense!